docusaurus/docs/reactnative/additional_resources.mdx

```diff
@@ -0,0 +1,55 @@
+---
+id: additional-resources
+sidebar_position: 8
+title: Additional Resources
+---
+
+Here's a list of resource links that could be helpful during development while using the Stream Chat React Native SDK library.
+
+## Internal Resources
+
+### [Stream Chat React Native GitHub](https://github.com/GetStream/stream-chat-react-native)
+
+### [Stream Chat React SDK Native - Demos](https://github.com/GetStream/react-native-samples)
+
+### [Stream Chat React Native SDK Tutorial](https://getstream.io/chat/react-native-chat/tutorial/)
+
+### [Stream JS Client](https://getstream.io/chat/docs/javascript/?language=javascript)
+
+### [Stream Chat API tutorial](https://getstream.io/chat/get_started/)
+
+## External Resources
```

Comment:
Maybe we have 3 sections, `Stream`, `Libraries`, `Technologies` or perhaps split it in a different manor, I think the TS and React are less valuable to many than the lib links, we should also find any other libs we are using like DayJS and include those.